orbit/pkg/platform/platform_windows.go

```diff
@@ -142,7 +142,7 @@ func SignalProcessBeforeTerminate(processName string) error {
 	return nil
 }
 
-// GetProcessByName gets a single running process object by its name.
+// GetProcessesByName gets a single running process object by its name.
 // Returns ErrProcessNotFound if the process was not found running.
 func GetProcessByName(name string) (*gopsutil_process.Process, error) {
```

Comment:
We'll also need to amend the Windows implementation to return all processes, right?

Reply:
Or document that on Windows we return only one for now (with a TODO in case there are issues in the future)

Reply:
It looks like it might cause issues, I'll see about making the Windows implementation return a list as well

Reply:
Added